packages/components/src/tab-panel/index.tsx

```diff
@@ -1,46 +1,39 @@
 /**
  * External dependencies
  */
-import classnames from 'classnames';
+import * as Ariakit from '@ariakit/react';
+import cx from 'classnames';
```

Comment:
I'm guessing this change to `cx` wasn't intentional?

Reply:
You're right... i have no idea how that happened! 6b98f0af2a